### DISCUSSION TITLE
Discussione riguardo all'avere più dialoghi che differiscono solo per il genere di alcune parole

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#50] [Generale]
Divagando...

si, che senso ha la dicitura "ragazza", "ragazzo", o "amico", "amica".
Insomma la necessità di distinguere "Nei testi" il giocante femminile da quello maschile.

Non fraintendetemi per carità, intendo nelle frasi scritte, quelle dei personaggi che interagisconno con il giocatore.
Nei sottotitoli etc..

Non si potrebbe trovare un escamotage? tipo al posto di ragazzo,ragazza o amica,amico, 
io avrei messo " Infante" per ogni occorrenza dove ci sia distinzione fra maschile, femminile. Sarebbe stato azzecato e unisex.
Ma sopratutto non si sarebbero dovute scrivere tutte quelle linee di testo doppie.

Ho visto, che solo in alcuni punti è necessario specificare il sesso, (quindi doppie linee), tipo nell'intro, dove dipende dal sesso del personaggio 
che abbiamo scelto all'inizio.

Perchè dico questo?

perchè anche se scelgo un personaggio femminile, quando inizio il tutorial con Smiling jack, lui mi dirà sempre "Che scena AMICO!".
"What a scene, man!"

In ogni caso Dice "Man" che noi abbiamo tradotto con "Amico\a", nei testi, perchè più consono alla situazione traduttoria.
e ancora, dice comunque "Kiddo", che abbiamo tradotto "ragazzo\a". sempre nei testi.

Vogliamo parlare del file audio in inglese?. Non ci sono doppie frasi per maschile e femminile come nei testi.

In ogni caso Jack si rivolge al maschile sempre e ogni volta.

In america questo videogame, lo giocavano senza sottotitoli, e comunque anche se sceglievi personaggio femminile.
il gioco ti parlava sempre e comunque al maschile.

ma vediamo un po se lavorando sui testi ...faccio un esempio:

"Haa ha ha haa ha What a scene, man! Hoo-wee! ha ha ha"
"Haa ha ha haa ha Che scena amico! Hoo-wee! ha ha ha"

e se invece fosse tradotta così?

"Haa ha ha haa ha Che scena! Hoo-wee! ha ha ha" (eliminando amico\amica).

oppure :

"Haa ha ha haa ha Che scena li dentro, he! Hoo-wee! ha ha ha". (aggiungendo qualcosina di consono).

anche qui :

"Ah... look, kiddo,"
"Ah....guarda, ragazzo,"

e se invece fosse tradotta così?

"Ah....guarda, Infante,"

...e così via.

Insomma,un escamotage traduttorio tutto Italiano per ovviare al problema una volta per tutte.

Fine Divagazione

**Comment by tesivo** [comment]
Concordo con te sul fatto che la traduzione possa prendersi la licenza di togliere quelle parole superflue che ci costringono ad avere due frasi identiche tranne per quella parola perché diversa nel genere.

Come hai appunto già detto, talvolta sono proprio differenti le due frasi, ed in tal caso ci sta l'avere entrambe le frasi.

Inoltre il doppiaggio è uno per frase se la frase è differente, questo è il motivo per cui talvolta il doppiaggio con il personaggio femminile non c'è, nel senso che non muove il labiale e non c'è audio.

Se ragionato bene potrebbe semplificare il processo di traduzione ed evitare i problemi di doppiaggio inglese con personaggio femminile. 

Diciamo che si potrebbe procedere in questo modo:
Visto che l'audio fa riferimento solo al personaggio maschile puntiamo alla traduzione completa per lui. 
Tenendo presente che, ogni qual volta nella frase in esame troviamo che andrebbe differenziata una parte della frase per il personaggio femminile, si cerca di evitare tale differenziazione (come hai fatto nell'esempio eliminando amico/amica). In  questo modo i casi in cui occorre sistemare il doppiaggio inglese (che per ora rimarrà comunque relativo al personaggio maschile) per il personaggio femminile diminuiscono.
Magari in un secondo momento potremmo: togliere le parole tolte anche dal doppiaggio (il "man" dell'esempio).


Riguardo agli esempi:
Preferirei puntare a questa : "Haa ha ha haa ha Che scena! Hoo-wee! ha ha ha" (eliminando amico\amica).

Anche se ci potrebbe stare anche questa, dopotutto è di medesimo significato: "Haa ha ha haa ha Che scena li dentro, he! Hoo-wee! ha ha ha". (aggiungendo qualcosina di consono).

---

Non mi convince infante, farei solo "Ah....guarda, "

Quindi sostanzialmente è uguale per la prima frase mentre farei senza "infante" nella seconda

**Reply by crazyLillo** [reply]
son daccordo con te, comunque teniamoci questa cosa per quanto ci sarà il tempo di farlo.
intanto continuamo quello che stiamo facendo.
il sasso io l'ho lanciato, vediamo...
